app/appRoutes.js

```diff
@@ -24,6 +24,63 @@ function appRoutes($stateProvider) {
     template: '<app-main></app-main>'
   };
 
+  var city = {
+    name: 'appMain.city',
+    abstract: true,
+    url: '^/city/:fips_id',
+    controller: require('./components/common/cityModule/cityController'),
+    template: require('./components/common/cityModule/index.html'),
+    resolve: {
+      city: function($stateParams, disclosureApi) {
+        return disclosureApi.locations.get({fips_id: $stateParams.fips_id});
+      }
+    },
+    data: {
+      moduleClasses: 'page',
+      pageClasses: 'city',
+      pageTitle: 'City',
+      pageDescription: 'Some description.'
+    }
+  };
+
+  var cityMoney = {
+    name: 'appMain.city.money',
+    url: '/money',
+    template: require('./components/common/cityModule/money/index.html'),
+    ncyBreadcrumb: {
+      label: '{{ city.location.name }}',
+      parent: 'appMain.city'
+    },
+    data: {
+      moduleClasses: 'page',
+      pageClasses: 'city',
+      pageTitle: 'City',
+      pageDescription: 'Some description.'
+    }
+  };
+
+  var cityElections = {
+    name: 'appMain.city.elections',
+    url: '/elections',
+    template: require('./components/common/cityModule/elections/index.html'),
+    controller: 'cityElectionsController',
+    ncyBreadcrumb: {
+      label: '{{ city.location.name }}',
+      parent: 'appMain.city'
+    },
+    resolve: {
+      ballot: function($stateParams, disclosureApi) {
+        return disclosureApi.ballot.get();
+      }
+    },
+    data: {
+      moduleClasses: 'page',
+      pageClasses: 'city',
+      pageTitle: 'City',
+      pageDescription: 'Some description.'
+    }
+  };
+
```

Comment:
I think these moved to [locality](https://github.com/caciviclab/disclosure-frontend/blob/master/app/components/appMainModule/localityPageModule/state.js#L12), but yeah, this is the right idea.